index.html

```diff
@@ -1688,47 +1693,62 @@ <h2>Client-Side Dereferencing</h2>
 		the <a>DID resolver</a>, and
 		<a href="#dereferencing-algorithm-fragment">Dereferencing the Fragment</a> is done by the <a>client</a>.</p>
 
-		<p>Example: Given the <a>DID URL</a> <code>did:xyz:1234#keys-1</code>, a <a>DID resolver</a> could be invoked
+		<section>
+			<h3>Example</h3>
+
+		<p>Given the <a>DID URL</a> <code>did:xyz:1234#keys-1</code>, a <a>DID resolver</a> could be invoked
 		via <a>local binding</a>
 		for <a href="#dereferencing-algorithm-resource">Dereferencing the Resource</a> (i.e., the <a>DID document</a>),
 		and the <a>client</a> could complete the <a>DID URL dereferencing</a> algorithm by
 		<a href="#dereferencing-algorithm-fragment">Dereferencing the Fragment</a> (i.e., a part of the <a>DID document</a>).</p>
 
 		<figure id="figure-client-side-dereferencing-1">
-			<img style="display: block; width: 100%;" src="diagrams/client-side-dereferencing-1.png"
+			<img style="display: block; width: 100%; margin-bottom: 20px;" src="diagrams/section-7/client-side-dereferencing-1.svg"
 			alt="Diagram showing client-side dereferencing of a DID URL by a DID resolver and a client">
 			<figcaption style="text-align: center;">
 				Client-side dereferencing of a <a>DID URL</a> by a <a>DID resolver</a> and a <a>client</a>.
 			</figcaption>
 		</figure>
 
-		<p>Example: Given the <a>DID URL</a> <code>did:xyz:1234#keys-1</code>, a <a>DID resolver</a> could be invoked via
-		<a>local binding</a> which invokes another <a>DID resolver</a> via <a>remote binding</a>
+		</section>
+
+		<section>
+			<h3>Example</h3>
```

Comment:
This could be more descriptive

Reply:
Actually looking at this again, I think we should rename the header. Currently in 7.2.3 there are three subheadings all of them Example.

Reply:
I think I'll update this PR to not use headers at all for these three examples, to avoid the entries in the ToC.